extract-patches.sh

```diff
@@ -2,6 +2,13 @@
 set -euo pipefail
 IFS=$'\n\t'
 
+getIndexedSubmoduleSha() {
+    (
+        cd "$DIR"
+        git ls-tree HEAD src/cli | awk '{print $3}'
```

Comment:
I don't understand why this is specific to cli?

Reply:
My bad, it's not. Fixing